apps/dashboard/components/logs/queries/queries-overflow-tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import type { LogsFilterUrlValue } from "@/app/(app)/logs/filters.schema";
+import { Tooltip, TooltipContent, TooltipTrigger } from "@unkey/ui";
+import { QueriesPill } from "./queries-pill";
+
+type QueriesOverflowProps = {
+  list?: LogsFilterUrlValue[];
+  listType?: "status" | "path" | "method";
+};
+
+export const QueriesOverflow = ({ list }: QueriesOverflowProps) => {
+  return list?.length ? (
```

Comment:
for readability, I think it would be nicer to return early and make it clear that if we don't have any elements to display, we just return null

it's not super important for this small component, but a good habit anyways.

```
if (!list || list.length === 0) {
  return null
}

return ...
```